app/models/case/sar/internal_review.rb

```diff
@@ -1,4 +1,11 @@
 class Case::SAR::InternalReview < Case::SAR::Standard
+
+  include LinkableOriginalCase
+
+  validates_presence_of :original_case
+
+  attr_accessor :original_case_number
+
```

Comment:
just for reminder, not forget to define the flag for timeliness and compliant sub type

Reply:
I will define this as part of the step after the confirmation step 👍